src/main/java/ai/promoted/delivery/client/PromotedDeliveryClient.java

```diff
@@ -7,18 +7,18 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import ai.promoted.delivery.model.ClientInfo;
-import ai.promoted.delivery.model.ClientType;
-import ai.promoted.delivery.model.CohortArm;
 import ai.promoted.delivery.model.CohortMembership;
 import ai.promoted.delivery.model.DeliveryExecution;
 import ai.promoted.delivery.model.DeliveryLog;
-import ai.promoted.delivery.model.ExecutionServer;
 import ai.promoted.delivery.model.LogRequest;
 import ai.promoted.delivery.model.Request;
 import ai.promoted.delivery.model.Response;
 import ai.promoted.delivery.model.Timing;
-import ai.promoted.delivery.model.TrafficType;
 import javax.annotation.Nullable;
+import ai.promoted.proto.event.CohortArm;
```

Comment:
We should get some sorta java linter in here.  It'd handle alphabetical sorting of imports.

Reply:
Yeah. I threw a bunch of Java extensions into my vscode and it didn't come with any kind of default preference here so I just left it